### PR TITLE
Run fuzz test with L0 when available

### PR DESCRIPTION
**Pull request by PatKamin**

### CODE REVIEW

**Review by lukaszstolarczuk — COMMENTED** [review]
👍

File: test/fuzz/CMakeLists.txt [hunk 1]
```diff
@@ -16,11 +16,19 @@ function(add_fuzz_test name label)
     add_test(NAME ${TEST_TARGET_NAME}
         COMMAND ${TEST_TARGET_NAME} ${ARGN} -verbosity=1
         WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+
+    set(ENV_VARS UR_ENABLE_LAYERS=UR_LAYER_FULL_VALIDATION)
+    if (UR_BUILD_ADAPTER_L0)
+        list(APPEND ENV_VARS
+                UR_ADAPTERS_FORCE_LOAD=\"$<TARGET_FILE:ur_adapter_level_zero>\"
+                NEOReadDebugKeys=1
```
Comment (pbalcer) [inline test/fuzz/CMakeLists.txt:24]: why?

**Review by pbalcer — APPROVED** [review]

File: test/fuzz/CMakeLists.txt (hunk 1 above)
Comment (PatKamin) [reply test/fuzz/CMakeLists.txt:24]: This combination of these two compute-runtime env vars is for dynamic linking of an Intel driver library without DEEPBIND. The error I get when running the test with sanitizers:
```
You are trying to dlopen a libigc.so.1 shared library with RTLD_DEEPBIND flag which is incompatible with sanitizer runtime (see https://github.com/google/sanitizers/issues/611 for details). If you want to run libigc.so.1 library under sanitizers please remove RTLD_DEEPBIND from dlopen flags.
```

Comment (pbalcer) [reply test/fuzz/CMakeLists.txt:24]: Ah, ok, I misunderstood what "read" here means :D

**Pull request merged**